components/mailer.rst

```diff
@@ -16,8 +16,129 @@ Installation
 
 .. include:: /components/require_autoload.rst.inc
 
+
+Introduction
+------------
+
+
 Usage
 -----
 
-We're currently working on the documentation of this component that was just
-added to Symfony. We'll publish it in a few days.
```

Comment:
We need some text above this that ends with a `::` - like:

```rst
The Mailer component has two main classes: a "transport" and the ``Mailer`` itself::
```

Reply:
I am not sure if you made a typo for "transport" . Or you were looking for ``Transport`` . As the mailer uses 2 single quotes, I have used it for Transport also.